saq/modules/email.py

```diff
@@ -1380,9 +1380,9 @@ def analyze_rfc822(self, _file):
 
         if 'from' in target_email:
             email_details[KEY_FROM] = decode_rfc2822(target_email['from'])
-            
+
             address = normalize_email_address(email_details[KEY_FROM])
-            if address != '':
+            if address is not None:
```

Comment:
Might want to just say
if address
here so that both != '' and is not None are taken in account?

Reply:
Yea I should have done that. I was surprised it wasn't like that before.